### DISCUSSION TITLE
你好你WSA项目失效了

### DISCUSSION THREAD

**Discussion by KaedeNashi** [LSPosed/LSPosed#2086] [Q&A]
你好你的WSA失效了，前两天我在试图重新打包安装带面具的wsa的时候打开GitHub发现已经没了，出现的是以下提示：
This repository has been disabled.
Access to this repository has been disabled by GitHub Staff due to a violation of GitHub's terms of service. If you are the owner of the repository, you may reach out to GitHub Support for more information.

**Comment by Howard20181** [comment]
https://github.com/LSPosed/MagiskOnWSALocal
